Sming/rboot/eagle.rom.addr.v6.ld

```diff
@@ -0,0 +1,347 @@
+PROVIDE ( Cache_Read_Disable = 0x400047f0 );
```

Comment:
@raburton  
I cannot apply the patch with the main repo /feature/rboot  (fa66616610)
I guess it is to be applied in your repo but commits hash doesn't seem to match.
Please let me know the reference repo and commit hash. 

Thanks